MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -861,6 +859,9 @@ public BandwidthMetricData onLoadStarted(long mediaStartTimeMs, long mediaEndTim
     public BandwidthMetricData onLoadCompleted(String segmentUrl, long bytesLoaded,
         Format trackFormat) {
       BandwidthMetricData segmentData = loadedSegments.get(segmentUrl);
+      if (segmentData == null) {
+        return null;
+      }
```

Comment:
it's definitely best that we don't blow up here, but do we actually expect this to happen? If so, are we missing any data (like does this happen ever for non-init fragment requests?

Reply:
I have not seen this happened so far, but if it does we then just ignore that segment, alternative would be to send segment with partial data, so at this point we missed onLoad data which are:
REQUEST_RESPONSE_START, REQUEST_MEDIA_START_TIME. 
If it make sense to submit the fragment without those data then we can do it but this may create a bad calculations in out Core SDK. 
RequestMetricsTracker expect REQUEST_RESPONSE_START to calculate loadTime and it uses loadTime to calculate throughput and average request throughput.

If you like I can create a new BandwidthMetricData object and continue the process.